internal/binutils/binutils_test.go

```diff
@@ -188,10 +188,9 @@ func skipUnlessDarwinAmd64(t *testing.T) {
 	}
 }
 
-func TestDisasm(t *testing.T) {
-	skipUnlessLinuxAmd64(t)
+func DisasmTestingInternal(t *testing.T, intelSyntax bool) {
 	bu := &Binutils{}
-	insts, err := bu.Disasm(filepath.Join("testdata", "exe_linux_64"), 0, math.MaxUint64)
+	insts, err := bu.Disasm(filepath.Join("testdata", "exe_linux_64"), 0, math.MaxUint64, false)
```

Comment:
'false' should be intelSyntax

Reply:
Do you mean I should switch the behavior for `Disasm`, i.e. passing `false` to `Disasm` to indicate users want Intel syntax?

Reply:
Now fixed.